app/jobs/replication_check_job.rb

```diff
@@ -0,0 +1,34 @@
+# Confirms existence of PreservedCopy on an endpoint.
+# Confirms the MD5 checksum matches in database and s3.
+class ReplicationCheckJob < ApplicationJob
+  # This queue is never expected to be used. For example:
+  # ReplicationCheck.set(queue: :endpoint_check_us_west_2).perform_later(pc)
+  # ReplicationCheck.set(queue: :endpoint_check_us_east_1).perform_later(pc)
+  queue_as :replication_check
```

Comment:
what if the queue name here is 'override-this-queue'  and the usage of the class is part of hte class comment.